diff --git a/configuration/brazil-config/app/AWSForecastingPipeline.cfg b/configuration/brazil-config/app/AWSForecastingPipeline.cfg
index 9aff8e57..722efac7 100644
--- a/configuration/brazil-config/app/AWSForecastingPipeline.cfg
+++ b/configuration/brazil-config/app/AWSForecastingPipeline.cfg
@@ -77,7 +77,7 @@ Development.*.log4j2.rootLogger.appenderRefs+=(LOG_APPENDER, CONSOLE_APPENDER);
 *.*.QuerylogHelper.filename=$ROOT/var/output/logs/service_log;
 *.*.QuerylogHelper.timeZone=UTC;
 
-Development.*.awsforecasting.platform.credentials = "com.amazon.awsforecasting.platform.development.service.$USER";
+Development.*.awsforecasting.platform.credentials = "com.amazon.awsforecasting.platform.development.service.yunyao";
 Beta.us-west-2.awsforecasting.platform.credentials = "com.amazon.awsforecasting.platform.pipeline.credentials.beta.pdx";
 Beta.us-east-1.awsforecasting.platform.credentials = "com.amazon.awsforecasting.platform.pipeline.credentials.beta.iad";
 Gamma.us-east-1.awsforecasting.platform.credentials = "com.amazon.awsforecasting.platform.pipeline.credentials.gamma.iad";
@@ -87,7 +87,7 @@ Prod.us-east-1.awsforecasting.platform.credentials = "com.amazon.awsforecasting.
 Development.us-east-1.awsforecasting.platform.sqs.base.url = "https://sqs.us-east-1.amazonaws.com/294920241382/";
 Development.us-west-2.awsforecasting.platform.sqs.base.url = "https://sqs.us-west-2.amazonaws.com/328345778731/";
 
-Development.*.awsforecasting.platform.server.queue = "pipelineforecasting-$USER";
+Development.*.awsforecasting.platform.server.queue = "pipelineforecasting-yunyao";
 Beta.*.awsforecasting.platform.server.queue = "pipelineforecasting-Prod";
 Gamma.*.awsforecasting.platform.server.queue = "pipelineforecasting-Prod";
 Prod.*.awsforecasting.platform.server.queue = "pipelineforecasting-Prod";
