diff --git a/configuration/brazil-config/app/AWSForecastingPlatformService.cfg b/configuration/brazil-config/app/AWSForecastingPlatformService.cfg
index c151c88..8f45d2a 100644
--- a/configuration/brazil-config/app/AWSForecastingPlatformService.cfg
+++ b/configuration/brazil-config/app/AWSForecastingPlatformService.cfg
@@ -52,9 +52,9 @@
     maxReplyThreads = 100;
   };
 
-development.*.awsforecasting.platform.credentials = "com.amazon.awsforecasting.platform.development.service.$USER";
+development.*.awsforecasting.platform.credentials = "com.amazon.awsforecasting.platform.development.service.yunyao";
 development.*.awsforecasting.platform.encryption = "com.amazon.awsforecasting.platform.development.encryption.pdx";
-development.*.awsforecasting.platform.bucket = "awsforecastingplatform-$REALM-$USER";
+development.*.awsforecasting.platform.bucket = "awsforecastingplatform-$REALM-yunyao";
 
 beta.us-west-2.awsforecasting.platform.encryption = "com.amazon.awsforecasting.platform.development.encryption.pdx";
 beta.us-east-1.awsforecasting.platform.encryption = "com.amazon.awsforecasting.platform.development.encryption.iad";
@@ -88,7 +88,7 @@ gamma.*.awsforecasting.platform.sdcVersion = "TWO";
 
 # development
 development.*.EventsLogger.CloudWatchEvents.Enabled = true;
-development.*.EventsLogger.CloudWatchEvents.OdinMaterial = "com.amazon.awsforecasting.platform.development.service.$USER";
+development.*.EventsLogger.CloudWatchEvents.OdinMaterial = "com.amazon.awsforecasting.platform.development.service.yunyao";
 development.*.EventsLogger.CloudWatchEvents.Endpoint = "https://jetstream-gamma-us-east-1-b.amazonaws.com";
 
 # beta
